packages/trader/src/App/Components/Elements/ContractAudit/contract-audit.tsx

```diff
@@ -1,35 +1,49 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { Tabs } from '@deriv/components';
 import { localize } from '@deriv/translations';
 import { WS } from '@deriv/shared';
+import { ContractUpdateHistory } from '@deriv/api-types';
+import { useTraderStore } from 'Stores/useTraderStores';
 import ContractDetails from './contract-details';
-import ContractHistory from './contract-history.jsx';
+import ContractHistory from './contract-history';
+
+type TContractUpdateHistory = [] | ContractUpdateHistory;
```

Comment:
```suggestion
type TContractUpdateHistory = TContractStore['contract_update_history'];
```
@kate-deriv  I guess '@deriv/api-types' are not installed in trader package.
but this type will be accessible from `TContractStore` in '@deriv/shared' after you merge my PR https://github.com/kate-deriv/deriv-app/pull/22/files#diff-361c2d590e3990a4dea8f8c724fbb23bca1c51236e53ed7b881a2d56f80f648a
You can also add this `contract_update_history` property to TContractStore here in your PR in order to use it before my PR is merged though :)
same for Line 13

Reply:
I think your PR will me merged faster, so I'll apply this suggestion after I updated parent branch